virt_lightning/api.py

```diff
@@ -184,16 +186,18 @@ def start(configuration, context="default", enable_console=False, **kwargs):
     if enable_console:
         import time
 
+        if console_fd is None:
+            console_fd = sys.stdout
+
         time.sleep(4)
         stream = conn.newStream(libvirt.VIR_STREAM_NONBLOCK)
         console = domain.dom.openConsole(None, stream, 0)
 
-        loop = kwargs.get("loop") or asyncio.get_event_loop()
-        _register_aio_virt_impl(loop)
+        _register_aio_virt_impl(loop=kwargs.get("loop"))
```

Comment:
kwargs.get("loop") can return None

Reply:
Which is fine, in this case `_register_aio_virt_impl()` get the loop with `asyncio.get_event_loop()`.